client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -317,23 +317,31 @@ class PreviewFrame extends React.Component {
 
   renderSketch() {
     const doc = this.iframeElement;
+    const localFiles = this.injectLocalFiles();
     if (this.props.isPlaying) {
-      srcDoc.set(doc, this.injectLocalFiles());
+      srcDoc.set(doc, localFiles);
       if (this.props.endSketchRefresh) {
         this.props.endSketchRefresh();
       }
     } else {
       doc.srcdoc = '';
       srcDoc.set(doc, '  ');
     }
+    const matches = localFiles.match(/createCanvas\((\d+), (\d+)\)/);
+    doc.width = `${matches[1]}px`;
+    doc.height = `${matches[2]}px`;
   }
 
   render() {
     return (
       <iframe
+        id="canvas_frame"
         className="preview-frame"
+        width="720px" // TODO
```

Comment:
do these values (width and height) have to be set? did you mean to return to this?

Reply:
Oh yeah, these were only temporary and I forgot to remove them. My bad !